csharp/Ice/CustomError/slice/Greeter.ice

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+module VisitorCenter
+{
+    /// The errors returned by the Greeter.
```

Comment:
The greeter does not return these enumerators. This enum is a field of GreeterException.